packages/hebao_v1/contracts/modules/transfers/TransferModule.sol

```diff
@@ -81,14 +81,15 @@ abstract contract TransferModule is BaseTransferModule
         address        token,
         address        to,
         uint           amount,
-        bytes calldata logdata
+        bytes calldata logdata,
+        bool           isWhitelisted
```

Comment:
with this parameter, the caller (the one who steals a wallet) can get around the quota check, so the security will be compromised...

Reply:
The logic doesn't just trust the value passed into the function. 

- `isWhitelisted == false` -> skip onchain whitelist check and **always** update the quota (unless disabled).
- `isWhitelisted == true` -> do onchain whitelist check and update the quota when not whitelisted.

So I think it's fine if I implemented the logic correctly. `if (!isWhitelisted || !isAddressDappOrWhitelisted(wallet, to)) { ... }`.

Maybe a better name would be `checkWhitelisted`?

Reply:
I see the point. Name it as `forceUsingQuota` or something like that is better.

Reply:
I like the logic, quite smart :)